extensions/mssql/src/contracts.ts

```diff
@@ -1044,27 +1044,28 @@ export namespace GetSqlMigrationAssessmentItemsRequest {
 
 export interface TableDesignerEditRequestParams {
 	tableInfo: azdata.designers.TableInfo,
-	tableChangeInfo: azdata.designers.DesignerEdit,
-	viewModel: azdata.designers.DesignerViewModel
+	tableChangeInfo: azdata.designers.DesignerEdit
 }
 
-export interface SaveTableDesignerChangesRequestParams {
-	tableInfo: azdata.designers.TableInfo,
-	viewModel: azdata.designers.DesignerViewModel
-}
-
-export namespace GetTableDesignerInfoRequest {
-	export const type = new RequestType<azdata.designers.TableInfo, azdata.designers.TableDesignerInfo, void, void>('tabledesigner/gettabledesignerinfo');
+export namespace InitializeTableDesignerRequest {
+	export const type = new RequestType<azdata.designers.TableInfo, azdata.designers.TableDesignerInfo, void, void>('tabledesigner/initialize');
 }
 
 export namespace ProcessTableDesignerEditRequest {
 	export const type = new RequestType<TableDesignerEditRequestParams, azdata.designers.DesignerEditResult, void, void>('tabledesigner/processedit');
 }
 
-export namespace SaveTableDesignerChangesRequest {
-	export const type = new RequestType<SaveTableDesignerChangesRequestParams, void, void, void>('tabledesigner/savechanges');
+export namespace PublishTableDesignerChangesRequest {
```

Comment:
were you planning on coming back and updating the "save" names (here and in other places in this PR) later? It's even more confusing now to have them mixed like this.

Reply:
yes, I'll do the rename in a separate PR.